connectors/tests/ftest.sh

```diff
@@ -14,12 +17,25 @@ sleep 30
 $ROOT_DIR/bin/fake-kibana --index-name search-$NAME --service-type $NAME --debug
 
 make load-data
+if [ $MEMPROFILE == $PERF_METRICS ]  && [ $PERF_METRICS == "yes" ]
+then
+    $ROOT_DIR/bin/perf8 --psutil --memray -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
+
+elif [ $MEMPROFILE == "yes" ]
+then
+    $ROOT_DIR/bin/perf8 --memray -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
+
+elif  [ $PERF_METRICS == "yes" ] 
+then
+    $ROOT_DIR/bin/perf8 --psutil -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
```

Comment:
we can now use:

```
$ROOT_DIR/bin/perf8 --memray --pyspy --psutil -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
```

Reply:
`--pyspy` gives you code profiling with no extra overhead. it works only on linux. it generates in the report dir a speedscope file yo can then load in https://speedscope.app

Reply:
Alright, I've updated the `ftest.sh` as per your suggestion 👍